CHANGELOG.md

```diff
@@ -7,10 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [v6.0.0] - 2021-05-03
```

Comment:
Update the `compare` link at the bottom of this file.
https://github.com/nasa/cumulus-dashboard#5-update-changelogmd

Reply:
Oof I did a bad job on this today

Reply:
9e82dbb